slmsuite/hardware/cameras/camera.py

```diff
@@ -343,9 +816,9 @@ def autofocus(self, z_get, z_set, z_list=None, plot=False):
 
         Parameters
         ----------
-        z_get : lambda
```

Comment:
feel like this should be get_z in analogy to the other get functions

Reply:
Fixed in e6804a2b9267bbd336cf14b037ccfed85a13f2f9, thanks!